content/docs/source-git/_index.md

```diff
@@ -4,6 +4,73 @@ draft: false
 weight: 1
 ---
 
+Let's talk about what source-git actually is.
+
+We initiated source-git as an alternative to working in
+[dist-git](https://github.com/release-engineering/dist-git). The need was
+especially amplified after we realized that many projects already do this. The
+problem was that everyone created a workflow and tooling for themselves which
+made it really hard to share knowledge and onboard new projects. Our intention
+is to create an implementation which is highly configurable, easy to start with
+and provdes both command-line tooling and a service which is tightly
+integration with GitHub and GitLab.
+
+Source-git is a git repository which contains upstream sources and packaging
+files for a downstream distribtions, in our case Fedora Linux, CentOS Stream
+and Red Hat Enterprise Linux.
+
+The basic requirement on a source-git repo is that one needs to be able to
+create a SRPM out of a current checkout. We need several steps to achieve such
```

Comment:
```suggestion
create a SRPM out of a current checkout. We need several files to achieve such
```

Reply:
not sure about this one though it's true your suggestion is more concrete, mine is abstract